pkg/frontend/mysql_cmd_executor.go

```diff
@@ -910,20 +911,20 @@ func (mce *MysqlCmdExecutor) handleCmdFieldList(tableName string) error {
 
 			names := db.Relations(txnHandler.GetTxn().GetCtx())
 			for _, name := range names {
-				table, err := db.Relation(name, txnHandler.GetTxn().GetCtx())
+				_, err := db.Relation(name, txnHandler.GetTxn().GetCtx())
 				if err != nil {
 					return err
 				}
-				defs := table.TableDefs(txnHandler.GetTxn().GetCtx())
-				for _, def := range defs {
-					if attr, ok := def.(*engine.AttributeDef); ok {
-						// Fixme: attrs here is never used.
-						attrs = append(attrs, &engineColumnInfo{
-							name: attr.Attr.Name,
-							typ:  attr.Attr.Type,
-						})
-					}
-				}
+				// Fixme: attrs here is never used.
```

Comment:
CMD_FIELD_LIST may be triggered sometimes. I do not recommend to comment the code.

Reply:
these code seems do nothing. because the next code will reset attrs.


cols, ok := mce.tableInfos[tableName]
	if !ok {
		//just give the empty info when there is no such table.
		attrs = make([]ColumnInfo, 0)
	} else {
		attrs = cols
	}

Reply:
I will fix it in my next pr .

Reply:
get it